templates/zerver/help/include/congrats-indented.md

```diff
@@ -0,0 +1,3 @@
+    **Congratulations! You're done!**
+
+    Your {{ integration_display_name }} notifications may look like:
```

Comment:
I don't think we need an indented version of this macro. This macro isn't a part of any numbered step and it's usage shouldn't be indented in any case, since the congrats message and the screenshot that follows isn't indented in any of our integrations' docs. :)